src/components/TransactionStatus.tsx

```diff
@@ -127,7 +127,7 @@ function TransactionStatus({ transaction, onComplete }: Props) {
       },
       {
         chainType: sourceChain,
-        label: 'Finalise block',
+        label: 'Finalize block',
```

Comment:
Are we using UK English or USA English?

Reply:
AFAIR "Google English" was the rule, but it's more skewed towards US obviously.